### DISCUSSION TITLE
Amazon FSx OpenZFS File System

### DISCUSSION THREAD

**Discussion by hoque93** [crossplane/crossplane#5123] [General]
I am trying to create an Amazon FSx OpenZFS File system but it seems Crossplane does not support this. Lustre, Windows FIle System and NetApp ONTAP are all available but OpenZFS is not: https://marketplace.upbound.io/providers/upbound/provider-aws-fsx/v0.46.0
Is there any reason this is the case? Are there any workarounds for this that will allow me to deploy with Crossplane?

**Comment by jbw976** [comment]
Good question! I think it just hasn't been prioritized yet. I see it listed in the not tested/supported resources still:
https://github.com/upbound/provider-aws/blob/main/config/externalnamenottested.go#L119-L120

Looks like there is a tracking issue for this, feel free to add a comment to express your need there too! :)

https://github.com/upbound/provider-aws/issues/344
